### PR TITLE
tests: Add fuzzing harness for {Read,Write}{LE,BE}{16,32,64} (crypto/common.h)

### PR DESCRIPTION
**Pull request by practicalswift**
Add fuzzing harness for `{Read,Write}{LE,BE}{16,32,64}` (`crypto/common.h`).

See [`doc/fuzzing.md`](https://github.com/bitcoin/bitcoin/blob/master/doc/fuzzing.md) for information on how to fuzz Bitcoin Core. Don't forget to contribute any coverage increasing inputs you find to the [Bitcoin Core fuzzing corpus repo](https://github.com/bitcoin-core/qa-assets).

Happy fuzzing :)

### CODE REVIEW

**Comment by MarcoFalke** [comment]
Any thoughts on clarifying that `ConsumeFixedLengthByteVector` returns a zero-filled byte vector when there remaining bytes are insufficient?

**Comment by practicalswift** [comment]
@MarcoFalke Good idea. Added!

File: src/test/fuzz/util.h [hunk 1]
```diff
@@ -214,4 +214,16 @@ NODISCARD inline bool ContainsSpentInput(const CTransaction& tx, const CCoinsVie
     return false;
 }
 
+// Returns a byte vector of size `length` regardless of the number of remaining bytes available
+// from the fuzzer. Pads with zero value bytes if needed to achieve a vector size of `length`.
```
Comment (MarcoFalke) [inline src/test/fuzz/util.h:218]: Sadly doxygen is disabled on the test directory, but for good karma I suggest to still use doxygen comments (see dev notes for example formattings)

Comment (practicalswift) [reply src/test/fuzz/util.h]: Fixed!

**Comment by MarcoFalke** [comment]
ACK cf5b8f64b3fef053035fa11231601b79bfa53aff

**Pull request merged**